src/services/user.services.js

```diff
@@ -23,10 +23,19 @@ const changePassword = (password, confirmNewPassword) =>
     })
   );
 
+const resetPassword = (email) =>
+  client.post(
+    'rest-auth/password/reset/',
+    JSON.stringify({
```

Comment:
Will you stringify all your request? if so, you could handle this in your client instance overriding the  [transformRequest](https://github.com/axios/axios) method. A better way to go is define interceptors for your requests/responses, but I guess transformRequest is pretty enough if you only want to apply little changes like this.

Reply:
I just saw you are already using interceptors, just add JSON.stringify(...) there.

Reply:
Ok I'll change this in another PR